pisa/core/container.py

```diff
@@ -20,6 +20,7 @@
 from pisa.core.translation import histogram, lookup, resample
 from pisa.utils.log import logging
 
+from itertools import chain
```

Comment:
This is somewhat nitpicky, but eventually you'll find it useful for reading Python code: Start with `__future__` imports, then group imports from python builtins and the python standard library (`collections`, `itertools`, etc.), then imports from external libraries (`numpy`, etc.), then imports from the local package (`pisa`).